### ISSUE TITLE
 Add Indicator from the ARS return that it is waiting for "X" more results

### ISSUE THREAD

**Issue by TranslatorIssueCreator** [NCATSTranslator/Feedback#159]
The search results incremental display could be smoother. Searching for "What gene is downregulated by Insulin" initially returns only one result. After a few minutes I can click "Load new results" to show more, but the search continues and I can't get more until it completes (~10 minutes). If I can't get results as they come in (load new results multiple times), some indicator of how close it is to being done would be useful.

**Comment by sierra-moxon** [comment]
Feedback from the UI team:
This will likely be an architecture issue to address; the UI returns results as they are available.

**Comment by capasfield** [comment]
From TAQA - 
Not a simple fix for this issue - more complex. 
Move monitoring and measuring speed onto the architecture agenda and include this issue.

**Comment by cbizon** [comment]
I think that the long term issue here is performance related.  So putting this in August, which is post-performance.

**Comment by cbizon** [comment]
Performance has been increased to within the assigned 5 minute window.  I think that this is currently a UI issue.

**Comment by Genomewide** [comment]
There is still no indication of how many results are coming. But they all get there more quickly. So, while I don't think it resolves it completely,  I also don't know what else can be done.   

We made the 'more results' indicator more interesting and it may distract people from the wait. But that sounds like 'I could not fix your breaks. So, I made your horn louder.- solution.  

@sierra-moxon I think we can close this?
